litmus-portal/backend/graphql-server/pkg/database/mongodb/init.go

```diff
@@ -18,8 +18,11 @@ var err error
 var collections = map[string]string{
 	"Cluster":     "cluster-collection",
 	"WorkflowRun": "workflow-run",
+	"User":        "user",
+	"Project":     "project",
 }
 
+var database *mongo.Database
```

Comment:
We are not reusing this anywhere, we can just have the collection variables.

Reply:
But if we do not have this. We will have to call `database.Name()` every time for initializing collections and we do have a number of collections for this code to be repeated